app/controllers/suggestions.rb

```diff
@@ -0,0 +1,142 @@
+get '/suggestions' do
+  @suggestions = Suggestion.all
+  @sorted_suggestions = @suggestions.sort_by{|suggestion| suggestion.votes.count}.reverse
```

Comment:
There is a SQL way to accomplish this sort (and thus an ActiveRecord way to accomplish this). Bonus points to the first Golden Bear to figure it out! (Why this matters: if the database was larger, the SQL sort would be _much faster_ than the Ruby memory sort.)